app/components-react/shared/inputs/TagsInput.tsx

```diff
@@ -28,9 +31,17 @@ export type TTagsInputProps<TValue> = TSlobsInputProps<
 >;
 
 export const TagsInput = InputComponent(<T extends any[]>(p: TTagsInputProps<T>) => {
-  const { inputAttrs, wrapperAttrs, forceUpdate } = useInput('tags', p);
+  const { inputAttrs, wrapperAttrs } = useInput('tags', p);
   const options = p.options;
   const tagsMap = useMemo(() => keyBy(options, 'value'), [options]);
+  const { s, updateState } = useFormState({ isAnimating: false });
+
+  async function animateWarning() {
+    if (s.isAnimating) return;
+    updateState({ isAnimating: true });
+    await Utils.sleep(1000);
```

Comment:
this animation takes a full second? that seems like way too long for UX

Reply:
It takes less than a seccond. It just doesn't allow to start animation again within 1s